Added customization to die

LaurenBassett opened the pull request:
Since advantage/disadvantage is not just DnD specific, I added three lines to allow the user to input the type of die they would like to use. This could be useful for other systems like Mage the Ascension or Fate.

LaurenBassett committed:
Added customization to die

Since advantage/disadvantage is not just DnD specific, I added three lines to allow the user to input the type of die they would like to use. This could be useful for other systems like Mage the Ascension or Fate.

alonzi reviewed (changes requested):
i like where this is headed, however we're gonna take it to the next level.

instead of doing a call of input, let's make it a command line argument. I will make an issue and assign it to you, stay tuned

alonzi commented:
Ok, I have created an issue and linked it to this pull request. We are taking the suggestion from @LaurenBassett and running with it.

LaurenBassett committed:
Changed to command line arg

I also tested edge cases and added a try/except to catch type errors. You should be able to type in anything and it will default to a d20 if confused. I set a cap at a d100 since there is not a die larger than that.
